packages/flame_tiled/example/lib/main.dart

```diff
@@ -12,16 +12,16 @@ void main() {
 class TiledGame extends FlameGame {
   late TiledComponent mapComponent;
 
-  final world = World();
-  late final CameraComponent cameraComponent;
+  TiledGame()
+      : super(
+          cameraComponent: CameraComponent.withFixedResolution(
+            width: 16 * 28,
+            height: 16 * 14,
+          ),
+        );
 
   @override
   Future<void> onLoad() async {
-    cameraComponent = CameraComponent.withFixedResolution(
-      world: world,
-      width: 16 * 28,
-      height: 16 * 14,
-    );
     cameraComponent.viewfinder.zoom = 0.5;
```

Comment:
would it make more sense to move these parameters to the constructor as well? either with `..` or by adding explicitly